services/api/src/resources/env-variables/resolvers.ts

```diff
@@ -311,6 +312,11 @@ export const addOrUpdateEnvVariableByName: ResolverFn = async (
     throw new Error('A variable name must be provided.');
   }
 
+  if (scope === 'internal_container_registry') {
+    logger.warn('Variable scope "internal_container_registry" is deprecated & can no longer be set.');
+    return;
```

Comment:
`logger.warn` only sends a message to stdout/container logs. If we throw an error instead, the API caller will see it too.

```suggestion
    throw new Error('Variable scope "internal_container_registry" is deprecated & can no longer be set.');
```

Same change needed in `addEnvVariable` too.